src/masoniteorm/commands/SeedRunCommand.py

```diff
@@ -15,13 +15,26 @@ class SeedRunCommand(Command):
     """
 
     def handle(self):
+
+        seeder = Seeder(dry=self.option("dry"), seed_path=self.option("directory"))
+
         if self.argument("table") == "None":
-            return Seeder(
-                dry=self.option("dry"), seed_path=self.option("directory")
-            ).run_database_seed()
+            seeder.run_database_seed()
+            seeder_seeded = "Database Seeder"
+
+        else:
+
+            table = self.argument("table")
```

Comment:
I find it simplier to just do (even if there is one repetition)
```suggestion
            table = self.argument("table")
            seeder_file = f"{underscore(table)}_table_seeder.{camelize(table)}TableSeeder"
            seeder.run_specific_seed(seeder_file)
            seeder_seeded = f"{camelize(table)}TableSeeder"
```